cypress/e2e/api.cy.ts

```diff
@@ -34,7 +34,7 @@ describe('API Test', () => {
       cy.get('.api-group li').each(($el) => {
         const anchor = $el.find('a');
         const text = anchor.text();
-        const link = anchor.attr('href').split('#')[0];
+        const link = anchor.attr('href')!.split('#')[0];
```

Comment:
I mena we wouldn't really activate this just to work around it again, would we?

```suggestion
        const link = anchor.attr('href')?.split('#')[0] ?? '';
```

Reply:
```suggestion
        const link = anchor.attr('href')?.split('#')[0] ?? 'MISSING';
```